### DISCUSSION TITLE
Can Sylius install to subdirectory of domain (nginx)?

### DISCUSSION THREAD

**Discussion by AmtechInnovarch** [Sylius/Sylius#12415] [Q&A] [answered]
I want to install Sylius to https://domain.com/store but everything I try breaks the app.

Is it possible to install Sylius to the /store subdirectory?

**Comment by Zales0123** [comment, marked as answer]
Hello @AmtechInnovarch! Sylius is a platform with which you have full freedom regarding both code and deployment. So yes, definitely it's possible to install Sylius in a subdirectory on the server (of course, it would require some configuration work from you).

Btw, please, use our [Slack](http://sylius.com/slack) for such questions - we're not using this discussion panel at all right now 🖖 :)

**Reply by AmtechInnovarch** [reply]
I'm not allowed to login to your slack. According to your slack I have to contact the workspace administrator at Sylius Developers for an invitation. I only need to know the setting name in Sylius for the site root to change it from `/` to `/store` and I will make the changes myself. Couldn't find that setting anywhere in the documentation.

**Comment by Rafikooo** [comment]
This discussion is being closed due to inactivity (4+ years).

  For subdirectory installation, the solution typically involves:
  - Web server configuration (nginx/apache) to serve from subdirectory
  - Symfony's routing configuration with base path
  - Proper asset configuration

  For current versions of Sylius, please refer to Symfony documentation on subdirectory installation or ask
  on our community channels.
